src/components/NavBar/NavBar.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import {
+  Menu,
+  MenuButton,
+  MenuList,
+  MenuItem,
+  Box,
+  Flex,
+  Text,
+  HStack,
+  Button,
+  Icon,
+} from '@chakra-ui/react';
+import { Globe } from 'react-feather';
+
+function NavBar() {
+  return (
+    <Box>
+      <Flex
+        justify="space-between"
+        align="center"
+        mx={16}
+        mt={3}
+        fonts="
+            Montserrat"
+        fontSize={16}
```

Comment:
Please use the proper theme token provided to you by chakra UI in which case it's fontSize='md'